src/app/common/__snapshots__/DataOverview.spec.ts.snap

```diff
@@ -57,7 +57,6 @@ exports[`DataOverview.vue renders additional scoped slot 1`] = `
   <tbody>
     
     <tr
-      class="is-selected"
```

Comment:
At first I was trying to fix the test to ensure this value remains but I realized that it was unintentionally there:

In the current implementation of DataOverview, when `props.selectedEntityName` is not provided, it will be the empty string. Then, in `getRowAttributes`, `entityName` is set to `props.selectedEntityName || props.tableData.data[0].name` which evaluates to `undefined` in this test as the table data doesn’t have a `name` property. And again since the table data doesn’t have a `name` property, the `name` from `getRowAttributes`’s first argument is _also_ `undefined` and so `name === entityName ? 'is-selected' : ''` evaluates to `'is-selected'` when that shouldn’t be the case.

That being said, I made some tweaks to how this state is produced to ensure that, as long as there is a `name` property`, a default row can always be marked as selected.